api/contacts/contacts.model.js

```diff
@@ -43,8 +43,24 @@ class Contact {
   constructor() {
     this.db = mongoose.model("contacts", contactSchema);
   }
-  getContacts = async () => {
-    return await this.db.find();
+
+  getContacts = async (query) => {
+    const { limit, page, sub, sort, ...otherQuery } = query;
+    const skipItems = (page - 1) * limit;
+    const subObject = sub
+      ? {
+          subscription: sub,
+        }
+      : {};
+
+    return await this.db
```

Comment:
move return await this.db
      .find({
        ...otherQuery,
        ...subObject,
      })
      .skip(skipItems)
      .limit(Number(limit))
      .sort({ name: 1 }); 


to pagination function